<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
<summary errors="1" failures="0" finished="true" name="Test Run: 0" tests="0" total_time="Finished in: 0.01 secs."/>
<test location="C:\Users\kumark\eclipse-workspace\PythonP\src\cfginit.py" status="error" test="cfginit" time="0">
<stdout/>
<stderr><![CDATA[Traceback (most recent call last):
  File "C:\Users\kumark\.p2\pool\plugins\org.python.pydev.core_6.5.0.201809011628\pysrc\_pydev_runfiles\pydev_runfiles.py", line 468, in __get_module_from_str
    mod = __import__(modname)
  File "C:\Users\kumark\eclipse-workspace\PythonP\src\cfginit.py", line 11, in <module>
    import yaml
ImportError: No module named yaml
ERROR: Module: cfginit could not be imported (file: C:\Users\kumark\eclipse-workspace\PythonP\src\cfginit.py).
]]></stderr>
</test>
<launch mode="run">
<launch_memento><![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="PythonP cfginit.py"/>
]]></launch_memento>
</launch>
</pydev-testsuite>
